server/app/views/applicant/ApplicantCommonIntakeUpsellTemplate.html

```diff
@@ -0,0 +1,57 @@
+<html th:attr="lang=${preferredLanguage.code()}">
+  <head
+    th:replace="~{applicant/ApplicantBaseFragment :: pageHeaderScriptsAndLinks}"
+  ></head>
+  <body data-load-modal="true">
+    <div th:replace="~{applicant/NavigationFragment :: pageHeader}"></div>
+    <main role="main">
+      <h1
+        th:text="#{title.commonIntakeConfirmationTi}"
+        th:if="${isTrustedIntermediary}"
+      ></h1>
+      <h1
+        th:text="#{title.commonIntakeConfirmation}"
+        th:if="${!isTrustedIntermediary}"
+      ></h1>
+
+      <div th:if="${eligiblePrograms.size()==0}" style="margin-bottom: 20px">
+        <p
+          th:utext="#{content.commonIntakeNoMatchingProgramsTi(${moreResourcesLinkHtml})}"
+          th:if="${isTrustedIntermediary}"
+        ></p>
+        <p
+          th:utext="#{content.commonIntakeNoMatchingPrograms(${moreResourcesLinkHtml})}"
+          th:if="${!isTrustedIntermediary}"
+        ></p>
+      </div>
+
+      <div th:if="${eligiblePrograms.size()>0}" style="margin-bottom: 20px">
```

Comment:
I believe we aren't actually supposed to use > and < in thymeleaf 😢 https://www.thymeleaf.org/doc/tutorials/2.1/usingthymeleaf.html#comparators-and-equality